exercises.md

```diff
@@ -19,7 +19,7 @@ The goal of this exercise is to practice to get a first impression of code and t
 
 #### (1 min) Independently - Glance at the code
 
-It's important that is an immediate reaction.
+It's important that what you use is your  immediate reaction, don't overthink it!
```

Comment:
Great!

At some point we should sync this doc with my runner slide data - I could probably use the markdown from here quite easily to populate https://github.com/CodeReadingClubs/runner/blob/main/src/Exercises.elm

Reply:
can we do that automatically? Would be cool!

Reply:
Yeah - would be good. I don't think it'd be too tricky.